test/functional/driver/driver-e2e-specs.js

```diff
@@ -83,30 +83,6 @@ describe('XCUITestDriver', function () {
         simsAfter.should.equal(simsBefore);
       });
 
-      it('with udid: uses sim and shuts it down afterwards', async () => {
```

Comment:
These assertions seem fine to me. It tests that when given a sim that already exists, Appium does not create a new one at any point.